opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -923,6 +923,7 @@ INSTANTIATE_PACK(std::map<std::pair<int,int>,std::pair<bool,double>>)
 INSTANTIATE_PACK(std::optional<double>)
 INSTANTIATE_PACK(std::optional<std::string>)
 INSTANTIATE_PACK(std::optional<GPMaint>)
+INSTANTIATE_PACK(std::optional<KeywordLocation>)
```

Comment:
this can't possibly work - there is no Mpi::packSize/Mpi::pack/Mpi::unpack for a KeywordLocation.